ui/helpers/constants/design-system.js

```diff
@@ -54,6 +54,7 @@ export const COLORS = {
   GOERLI: 'goerli',
   TRANSPARENT: 'transparent',
   LOCALHOST: 'localhost',
+  INHERIT: 'inherit',
```

Comment:
This should be taken care of in https://github.com/MetaMask/metamask-extension/pull/15507